packages/analytics/package.json

```diff
@@ -15,9 +15,10 @@
     "build": "rollup -c",
     "build:deps": "lerna run --scope @firebase/analytics --include-dependencies build",
     "dev": "rollup -c -w",
-    "test": "run-p lint test:browser",
-    "test:ci": "node ../../scripts/run_tests_in_ci.js -s test:browser",
+    "test": "run-p lint test:browser test:integration",
+    "test:ci": "node ../../scripts/run_tests_in_ci.js -s test",
```

Comment:
`test:ci` should not run lint. You can create a `test:all` script that doesn't do lint and to be used by `test` and `test:ci`

Reply:
Done.